resources/definitions/fdmprinter.def.json

```diff
@@ -5735,7 +5748,7 @@
                     "minimum_value": "0",
                     "minimum_value_warning": "raft_interface_line_width",
                     "maximum_value_warning": "15.0",
-                    "enabled": "resolveOrValue('adhesion_type') == 'raft'",
+                    "enabled": "resolveOrValue('adhesion_type') == 'raft' and raft_interface_layers > 0",
```

Comment:
`"enabled": "resolveOrValue('adhesion_type') == 'raft' and raft_interface_layers > 0"`

should be added to the following settings:

- "raft_interface_speed"
- "raft_interface_acceleration"
- "raft_interface_jerk"
- "raft_interface_fan_speed"
- "raft_interface_extruder_nr"